parastell/nwl_utils.py

```diff
@@ -400,9 +416,17 @@ def compute_nwl(
                 corners
             )
 
-    nwl_mat = nwl_mat / area_mat
+    # Normalize NWL statistics to area
+    nwl_avg /= area_mat
+    nwl_std_dev /= area_mat
```

Comment:
This math may not be right for the std_dev?

Reply:
Your math looks good to me. As demonstrated by your proof, the [standard deviation scales directly with an applied constant](https://en.wikipedia.org/wiki/Standard_deviation#Identities_and_mathematical_properties)

Reply:
If you'd prefer, I can divide the original data set by the area and then calculate the mean and standard deviation